packages/markdown-overrides/src/ErrorBlock/__tests__/ErrorBlock-test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { ErrorBlock } from '../';
+
+describe('ErrorBlock', () => {
+  it('renders itself and children', () => {
+    const errorText = 'error!';
+    const { getByText } = render(<ErrorBlock>{errorText}</ErrorBlock>);
+
+    getByText(errorText);
```

Comment:
This is the normal recommended approach for the RTL library and is very good. I've been sneakily pushing an alternative style in our repos because I think it's a little better ergonomically:

```suggestion
    const view = render(<ErrorBlock>{errorText}</ErrorBlock>);

    view.getByText(errorText);
```

Totally up to you what you want 😉 this is a comment, not a request for changes.